kubernetes-ts-multicloud/Pulumi.yaml

```diff
@@ -0,0 +1,11 @@
+name: kubernetes-ts-multicloud
+description: Multi-cloud Kubernetes example
+runtime: nodejs
+template:
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    azure:location:
+      description: The Azure location to deploy into
+      default: westus2
```

Comment:
I see settings for AWS and Azure, but not GKE. Is this expected?

Reply:
Good catch. Had it set in my ambient config.